docs/studies/covid-symptom.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Covid Symptom
+parent: Studies
+grand_parent: ETL
+nav_order: 1
+# audience: engineer familiar with the project
+# type: howto
+---
+
+# The Covid Symptom Study
+
+This study uses NLP to identify symptoms of COVID-19 in clinical notes.
+Specifically, it uses [cTAKES](https://ctakes.apache.org/) and
+[cNLP transformers](https://github.com/Machine-Learning-for-Medical-Language/cnlp_transformers)
+to identify clinical terms.
+
+## Preparation
+
+Because it uses external services like cTAKES, you will want to make sure those are ready.
+From your git clone of the `cumulus-etl` repo, you can run the following to run those services:
+```shell
+export UMLS_API_KEY=your-umls-api-key  # don't forget to set this - cTAKES needs it
+docker compose --profile covid-symptom-gpu up -d
+```
+
+You'll notice the `-gpu` suffix there.
+Running NLP is much, much faster with access to a GPU,
+so we strongly recommend you run this on GPU-enabled hardware.
+
+And since we _are_ running the GPU profile, when you do run the ETL,
+you'll want to launch the GPU mode instead of the default `cumulus-etl` CPU mode:
+```shell
+docker compose run cumulus-etl-gpu …
+```
+
+But if you can't use a GPU or you just want to test things out,
+you can use `--profile covid-symptom` above and the normal `cumulus-etl` run line to use the CPU.
+
+## Tasks
+
+There is one main task, run with `--task covid_symptom__nlp_results`.
+
+This will need access to clinical notes,
+which are pulled fresh from your EHR (since the ETL doesn't store clinical notes).
+This means you will likely have to provide some other FHIR authentication arguments like
+`--smart-client-id` and `--fhir-url`.
+See `--help` for more authentication options.
+
+There is also a second optional task `--task covid_symptom__nlp_results_term_exists`,
+which just uses a different polarity cNLP transformer (`termexists` rather than `negation`).
+You likely don't need both, but they may be interesting to compare.
```

Comment:
Q: do we want to treat this as a in development style command? i.e. maybe it doesn't live here until we've decided to use it in a non-experimental way?

Reply:
Good question.

So this task is not like, in-development or experimental in the sense of a WIP that's likely to change. Once this lands, I feel like it's a full, normal task. But it is "less than" a normal task because it's kind of a special case task to help Tim compare the new transformer's performance. But I guess all study tasks are kind of special case tasks like that.

But here's this task's current status in this PR:
1. is not run by default
2. can only be run by name (i.e. not tagged at all -- `--task-filter=covid_symptom` or `--task-filter=gpu` do not hit it)
3. is documented on the study's docs

Is your thinking that 1 and 2 are good (i.e. "lowlightling" this task) and 3 should also not mention the task to lowlight it even further?

Reply:
(I'm open to that, just trying to clarify what you are thinking about when marking experimental -- or do you want like active barriers like a flag `--enable-experimental` or something)

Reply:
i think in my head, i'm drawing the line at 'this is something we are using internally while doing study validation' versus 'this is something a partner site might be explicitly asked to run' - and I think this is mostly a 3 level comment.

Reply:
Fair yeah - I dropped mention from the docs :+1: